### PR TITLE
refactor "experimental" to "extensions" in documents

### PR DESCRIPTION
**Pull request by caesarxuchao**
ref #15378 #15409

@lavalamp @nikhiljindal 

I haven't fixed the api-changes.md, as that requires some real rewrite.

### CODE REVIEW

**Comment by nikhiljindal** [comment]
LGTM

File: docs/admin/daemons.md [hunk 1]
```diff
@@ -209,9 +209,9 @@ all or certain hosts, and when it needs to start before other pods.
 
 ## Caveats
 
-DaemonSet is part of the experimental API group, so it is not subject to the same compatibility
+DaemonSet is part of the `extensions` API group, so it is not subject to the same compatibility
```
Comment (lavalamp) [inline docs/admin/daemons.md]: "not subject to the same compatibility guarantees" needs fixing, but I don't have a concrete suggestion off the top of my head. possibly overlaps w/ @erictune's doc changes.

**Comment by wojtek-t** [comment]
@caesarxuchao please rebase

File: docs/admin/daemons.md (hunk 1 above)
Comment (erictune) [inline docs/admin/daemons.md]: yes, my change will reword.  lets just get this PR in.

**Comment by caesarxuchao** [comment]
Rebased. @lavalamp may I get the "lgtm" per Eric's comment https://github.com/kubernetes/kubernetes/pull/15517#discussion_r42141098? Thanks.

**Comment by lavalamp** [comment]
lgtm

**Comment by lavalamp** [comment]
@caesarxuchao looks like some tests need attention.

**Comment by caesarxuchao** [comment]
e2e is a flake:

```
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/container_probe.go:74
pod never became ready
Expected error:
    <*errors.errorString | 0xc20811ee70>: {
        s: "timed out waiting for the condition",
    }
    timed out waiting for the condition
not to have occurred
```

**Comment by caesarxuchao** [comment]
@k8s-bot test this please

**Comment by caesarxuchao** [comment]
add back "lgtm". it's a rebase.

**Pull request merged**